src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -310,6 +311,36 @@ or IAsyncDisposable
                 );
         }
 
-        return loggerSinkConfiguration.Sink(wrapper, restrictedToMinimumLevel, levelSwitch);
+        return wrapper;
+    }
+
+    /// <summary>
+    /// Helper method for constructing sinks outside of a logger pipeline.
+    /// </summary>
+    /// <param name="configure">An action that configures one or more sinks.</param>
+    /// <returns>If only a single sink is configured,
+    /// it will be returned from <see cref="CreateSink"/>. If zero or many sinks are configured, they will be combined
+    /// in an aggregating wrapper.</returns>
+    /// <exception cref="ArgumentNullException">When <paramref name="configure"/> is <code>null</code>.</exception>
+    public static ILogEventSink CreateSink(Action<LoggerSinkConfiguration> configure)
+    {
+        Guard.AgainstNull(configure);
+
+        var sinksToWrap = new List<ILogEventSink>();
+
+        var capturingConfiguration = new LoggerConfiguration();
+        var capturingLoggerSinkConfiguration = new LoggerSinkConfiguration(
+            capturingConfiguration,
+            sinksToWrap.Add);
+
+        // `WriteTo.Sink()` will return the capturing configuration; this ensures chained `WriteTo` gets back
+        // to the capturing sink configuration, enabling `WriteTo.X().WriteTo.Y()`.
+        capturingConfiguration.WriteTo = capturingLoggerSinkConfiguration;
+
+        configure(capturingLoggerSinkConfiguration);
+
+        return sinksToWrap.Count == 1 ?
+            sinksToWrap.Single() :
```

Comment:
perfectly cromulent code but the use of `Single`, rather than conveying intention is just distracting IMO
```suggestion
            sinksToWrap[0] :
```